GVFS/GVFS/CommandLine/UpgradeVerb.cs

```diff
@@ -128,6 +128,10 @@ private bool TryRunProductUpgrade()
             if (newestVersion == null)
             {
                 this.ReportInfoToConsole($"Great news, you're all caught up on upgrades in the {this.upgrader.Ring} ring!");
+
+                // Make sure there a no asset installers remaining in the Downloads directory. This can happen if user
+                // upgraded by manually downloading and running asset installers.
+                this.upgrader.DeletePreviousDownloads();

```

Comment:
Can you add functional test coverage for this scenario?

Reply:
Sure.